lib/voom/presenters/plugins/cacheable/component.rb

```diff
@@ -14,9 +15,13 @@ module Plugins
       module Cacheable
 
         class Component < DSL::Components::Base
+          include Mixins::CacheStore
           DSL::Components::Mixins.constants(false).each do |mixin|
             const = DSL::Components::Mixins.const_get(mixin)
-            include const if const.is_a?(Module)
+            if const.is_a?(Module)
+              include Mixins::CacheStore
```

Comment:
This group of changes isn't needed and just does some duplicate includes of `Mixins::CacheStore` which is already included on line 18.

Reply:
```suggestion
              include const if const.is_a?(Module)
```